### PR TITLE
Czech translation Update

### PR DESCRIPTION
**Pull request by joyinko**
Dwellings, Structures, Monsters

### CODE REVIEW

**Review by joyinko — COMMENTED** [review]
Corrected two typos.

**Comment by joyinko** [comment]
HI @zenseii , I corrected two problems I found. Could you please take a look as well? Thx

File: files/lang/cs.po [hunk 1]
```diff
@@ -6735,320 +6734,316 @@ msgid "Random Artifact - Major"
...
-msgstr ""
+msgstr "Oltář přivolání Země"
 
 msgid "Water Summoning Altar"
-msgstr ""
+msgstr "Oltář přivolání Vody"
 
 msgid "Unknown Monster"
-msgstr ""
+msgstr "Neznámá strvůra"
```
[the remark this answers is not in the document]
Comment (joyinko) [reply files/lang/cs.po]: Hi @zenseii , yes. It is just a typo. Late night work. :smile: 
Good catch, thanks.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@joyinko , thank you so much for the update of this translation!

**Pull request merged**